core/src/main/scala/kafka/tools/PerfConfig.scala

```diff
@@ -40,6 +40,10 @@ class PerfConfig(args: Array[String]) {
   val showDetailedStatsOpt = parser.accepts("show-detailed-stats", "If set, stats are reported for each reporting " +
     "interval as configured by reporting-interval")
   val hideHeaderOpt = parser.accepts("hide-header", "If set, skips printing the header for the stats ")
+  val helpOpt = parser.accepts("help", "Print usage.")
+}
+
+class LegacyProducerPerfConfig(args: Array[String]) extends PerfConfig(args) {
```

Comment:
Why do we need this class? Can you not simply move the code to `ProducerPerfConfig`?